form/src/main/java/org/dhis2/form/ui/binding/CoordinatesBindings.kt

```diff
@@ -166,6 +168,20 @@ fun TextInputEditText.setOnGeometryLongitudeEditorActionListener(
     }
 }
 
+@BindingAdapter(value = ["model", "loadingView"], requireAll = true)
+fun ImageButton.setOnLocationClick(model: FieldUiModel, progress: View) {
+    setOnClickListener {
+        this.visibility = View.GONE
+        progress.visibility = View.VISIBLE
+        model.invokeUiEvent(UiEventType.REQUEST_CURRENT_LOCATION)
+    }
+}
+
+@BindingAdapter("progressVisibility")
+fun View.progressVisibility(model: FieldUiModel) {
+    visibility = View.GONE
+}
```

Comment:
It's this binding adapter necesary? it is setting visibility and it's not using the model

Reply:
@mmmateos don't know why, but when I removed it it did not work as expected. I think it is on how binding adapters are handled